test.sh

```diff
@@ -14,8 +14,11 @@ end flake8
 
 start python
 pushd hubmap
-python manage.py test
+python3 manage.py test
 # TODO: Django tests
+service postgresql start
```

Comment:
This should not be necessary; Postgres is started by the `addons` configuration in `.travis.yml`.